src/agents/bee/runners/granite/runner.ts

```diff
@@ -83,6 +88,15 @@ export class GraniteRunner extends DefaultRunner {
     return memory;
   }
 
+  get templates(): BeeAgentTemplates {
```

Comment:
Now that we have this templates method, I wonder if we should be using this everywhere as opposed to using `this.input.templates`. If we use this templates method then we can removed the granite runner constructor code that duplicates this code.

https://github.com/i-am-bee/bee-agent-framework/blob/be3bb5c6d1a008cc205276b7ad0e6b759ae3ec97/src/agents/bee/runners/granite/runner.ts#L46C1-L48C84

Reply:
Yes, the template override is not necessary anymore. Removed.